src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final String FORMAT_DATE = "dd.MM.yyyy";
+    static final int POSITION_DATE_IN_ARRAY_DATA = 0;
+    static final int POSITION_NAME_IN_ARRAY_DATE = 1;
+    static final int POSITION_HOURS_IN_ARRAY_DATE = 2;
+    static final int POSITION_SALARY_IN_ARRAY_DATE = 3;
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromSalaryInfo = LocalDate.parse(dateFrom,
+                DateTimeFormatter.ofPattern(FORMAT_DATE));
+        LocalDate dateToSalaryInfo = LocalDate.parse(dateTo,
+                DateTimeFormatter.ofPattern(FORMAT_DATE));
+        LocalDate dateWorkUser;
```

Comment:
can we declare this variable later for better readability?
and let's use more informative name - `currentDate`